ports/serioussam-tfe/Serious Sam - The First Encounter.sh

```diff
@@ -41,18 +45,19 @@ else
   source "${controlfolder}/libgl_default.txt"
 fi
 
-if [ "$LIBGL_FB" != "" ]; then
-export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es/libGL.so.1"
-export LIBGL_ES=1
-export LIBGL_GL=14
-fi 
-
+export LD_LIBRARY_PATH="$GAMEDIR/Bin/libs:/usr/lib32:$LD_LIBRARY_PATH"
 export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
 
-$GPTOKEYB "ssam-tfe" -c "$GAMEDIR/serioussam.gptk" &
+export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es/libGL.so.1"
```

Comment:
We have the put the export SDL_VIDEO_GL_DRIVER in the IF statement else Panfrost will export the gl4es as a driver.